src/Microsoft.CSharp/src/project.json

```diff
@@ -1,30 +1,25 @@
 {
   "dependencies": {
-    "Microsoft.NETCore.Platforms": "1.0.1",
-    "System.Collections": "4.0.10",
-    "System.Diagnostics.Debug": "4.0.10",
-    "System.Diagnostics.Tools": "4.0.0",
-    "System.Dynamic.Runtime": "4.0.0",
-    "System.Globalization": "4.0.10",
-    "System.Linq": "4.0.0",
-    "System.Linq.Expressions": "4.0.0",
-    "System.ObjectModel": "4.0.10",
-    "System.Reflection": "4.0.10",
-    "System.Reflection.Extensions": "4.0.0",
-    "System.Reflection.Primitives": "4.0.0",
-    "System.Reflection.TypeExtensions": "4.0.0",
-    "System.Resources.ResourceManager": "4.0.0",
-    "System.Runtime": "4.0.20",
-    "System.Runtime.Extensions": "4.0.10",
-    "System.Runtime.InteropServices": "4.0.20",
-    "System.Threading": "4.0.10",
-    "System.Threading.Tasks": "4.0.10"
+    "System.Collections": "4.0.11",
+    "System.Diagnostics.Debug": "4.0.11",
+    "System.Diagnostics.Tools": "4.0.1",
+    "System.Dynamic.Runtime": "4.0.11",
+    "System.Globalization": "4.0.11",
+    "System.Linq": "4.1.0",
+    "System.Linq.Expressions": "4.1.0",
+    "System.ObjectModel": "4.0.12",
+    "System.Reflection": "4.1.0",
+    "System.Reflection.Extensions": "4.0.1",
+    "System.Reflection.Primitives": "4.0.1",
+    "System.Reflection.TypeExtensions": "4.1.0",
+    "System.Resources.ResourceManager": "4.0.1",
+    "System.Runtime": "4.1.0",
+    "System.Runtime.Extensions": "4.1.0",
+    "System.Runtime.InteropServices": "4.1.0",
+    "System.Threading": "4.0.11",
+    "System.Threading.Tasks": "4.0.11"
```

Comment:
Same question; I'm unclear when we choose to leave these as the original versions or update them.